sdk/tables/azure-data-tables/azure/data/tables/_error.py

```diff
@@ -95,111 +103,11 @@ def _to_str(value):
                             'Data was either not encrypted or metadata has been lost.'
 
 
-def _dont_fail_on_exist(error):
-    """ don't throw exception if the resource exists.
-    This is called by create_* APIs with fail_on_exist=False"""
-    if isinstance(error, ResourceExistsError):  # pylint: disable=R1705
-        return False
-    else:
-        raise error
-
-
-def _dont_fail_not_exist(error):
-    """ don't throw exception if the resource doesn't exist.
-    This is called by create_* APIs with fail_on_exist=False"""
-    if isinstance(error, ResourceNotFoundError):  # pylint: disable=R1705
-        return False
-    else:
-        raise error
-
-
-def _http_error_handler(http_error):
-    """ Simple error handler for azure."""
-    message = str(http_error)
-    error_code = None
-
-    if 'x-ms-error-code' in http_error.respheader:
-        error_code = http_error.respheader['x-ms-error-code']
-        message += ' ErrorCode: ' + error_code
-
-    if http_error.respbody is not None:
-        message += '\n' + http_error.respbody.decode('utf-8-sig')
-
-    ex = HttpResponseError(message, http_error.status)
-    ex.error_code = error_code
-
-    raise ex
-
-
-def _validate_type_bytes(param_name, param):
-    if not isinstance(param, bytes):
-        raise TypeError(_ERROR_VALUE_SHOULD_BE_BYTES.format(param_name))
-
-
-def _validate_type_bytes_or_stream(param_name, param):
-    if not (isinstance(param, bytes) or hasattr(param, 'read')):
-        raise TypeError(_ERROR_VALUE_SHOULD_BE_BYTES_OR_STREAM.format(param_name))
-
-
 def _validate_not_none(param_name, param):
     if param is None:
         raise ValueError(_ERROR_VALUE_NONE.format(param_name))
 
 
-def _validate_content_match(server_md5, computed_md5):
```

Comment:
is this chunk getting deleted because it's unused? wondering if we'll need this code to add support for encryption.

Reply:
Yes this is not used anywhere, I'll talk to @annatisch if it's needed for encryption

Reply:
This is not used in encryption. Anna approved the deletion.